hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMDBCheckpointServlet.java

```diff
@@ -216,12 +218,12 @@ public static Map<String, Map<Path, Path>> normalizeExcludeList(
             sstBackupDir.getOriginalDir().toString().length() + 1;
         Path srcPath = Paths.get(sstBackupDir.getTmpDir().toString(),
             truncateFileName(truncateLength, destPath));
-        paths.computeIfAbsent(fileName.toString(), (k) -> new HashMap<>()).put(srcPath, destPath);
+        paths.get(fileName.toString()).put(srcPath, destPath);
```

Comment:
Please assign result of `paths.computeIfAbsent` to a variable and use it instead of `paths.get(...)`.

(Sorry if this was not clear from previous review comment.)

Reply:
done